presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -590,7 +590,7 @@ def test_demo_text(unit_test_guid, nlp_engine):
         "Can you please check the status on bank account 954567876544 "
         "in PresidiBank?\n\n"
         ""
-        "Kate's social security number is 078-05-1120.  "
```

Comment:
https://www.ssa.gov/history/ssn/misused.html
this is a known mis-used SSN, it's not in circulation.